app/experimenter/experiments/forms.py

```diff
@@ -1271,10 +1272,42 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
+            ids = re.split(",", self.cleaned_data["other_normandy_ids"])
+            for id in ids:
```

Comment:
Technically `id` is a reserved word in python.  Python doesn't stop you from reusing reserved words but it's generally a good idea to avoid doing it.